src/ORM/Connect/MySQLDatabase.php

```diff
@@ -49,6 +49,12 @@ class MySQLDatabase extends Database
      */
     private static $charset = 'utf8';
 
+    /**
+     * Cache for getTransactionManager()
+     * @var TransactionManager
+     */
+    private $transactionManager = null;
```

Comment:
Protected over private?

Reply:
No, it's got a getter that subclasses should use.